nodes/src/nodes/llm_base/IInstance.py

```
@@ -21,13 +21,63 @@
+            with lock:
+                self.instance.sendSSE(
+                    'llm.chunk',
+                    nodeId=nodeId,
+                    seq=next(counter),
+                    text=text,
+                    done=done,
+                    error=error,
+                )
+
+        try:
+            answer = self.IGlobal._chat.chat_stream(question, lambda delta: emit(delta))
```

Ruff (0.15.12)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)